worker.go

```diff
@@ -11,14 +11,15 @@ import (
 
 // Worker is the doodad that does work.
 type Worker struct {
-	id       int
-	WorkChan chan UDPMetric
-	QuitChan chan struct{}
-	metrics  int64
-	mutex    *sync.Mutex
-	stats    *statsd.Client
-	logger   *logrus.Logger
-	wm       WorkerMetrics
+	id        int
+	WorkChan  chan UDPMetric
+	QuitChan  chan struct{}
+	processed int64
```

Comment:
I was debating that; personally I decided not to because the flush code path doesn't need those numbers. since the whole WorkerMetrics struct gets pushed out at flush time, I'd rather stick to fields that the flusher will use. might end up changing my mind later